### PR TITLE
Inline Solidus Frontend

### PR DESCRIPTION
**Pull request by gsmendoza**
This is actually part of https://github.com/gsmendoza/solidus_frontend/pull/1, but I've separated these commits from that PR to make the diff of that PR easier to view. For this PR, I'd recommend going through the individual commits instead of viewing the PR diff.

### CODE REVIEW

**Comment by kennyadsl** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -1,403 +1,42 @@
-<img width="250" src="./logo.svg" alt="Solidus logo">
+# solidus\_frontend
 
-# Solidus
+Frontend contains controllers and views implementing a storefront and cart for Solidus.
```
Comment (waiting-for-dev) [inline README.md:3]: I would add a notice to let new users know that this is not the recommended frontend anymore.

File: README.md [hunk 2]
```diff
@@ -1,403 +1,42 @@
...
-and change `vendor/assets/javascripts/spree/backend/all.js` as follows:
-
-```js
-//= require turbolinks
-//
-// ... current file content
-//
-//= require spree/backend/turbolinks-integration.js
-```
+## Testing
```
Comment (waiting-for-dev) [inline README.md:36]: We're not adding tests to the host application if I'm not wrong. So I think this section should instead be called "Contributing".

**Review by waiting-for-dev — COMMENTED** [review]
Thanks, @gsmendoza, I left some minor comments. Also:

- Could we reconfigure CI to be sure everything is well tied?
- -I think you can remove `.dockerdev/` and `docker-compose.yml` for now.

**Comment by waiting-for-dev** [comment]
> Could we reconfigure CI to be sure everything is well tied?

I saw your comment on #1, so I'm ok with doing that there.

**Comment by waiting-for-dev** [comment]
> I think you can remove .dockerdev/ and docker-compose.yml for now.

Ok, I also saw your checkbox about it in #1. So, if it's working, we can leave it. Did you try to run tests?

**Comment by waiting-for-dev** [comment]
I'll move those comments to #1, as it's where they belong.

**Review by waiting-for-dev — APPROVED** [review]

**Comment by gsmendoza** [comment]
> Ok, I also saw your checkbox about it in https://github.com/gsmendoza/solidus_legacy_frontend/pull/1. So, if it's working, we can leave it. Did you try to run tests?

Yep, I was able to get `docker-compose exec app bin/rspec` to pass :)

**Comment by gsmendoza** [comment]
Closing PR since #1 is more up-to-date.

**Comment by gsmendoza** [comment]
Reopening the PR because #2 is too difficult to comment on with its changes.

File: Gemfile [hunk 3]
```diff
@@ -3,7 +3,11 @@
 source 'https://rubygems.org'
 
 group :backend, :frontend, :core, :api do
-  gemspec require: false
+  gem 'solidus_api', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_backend', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_core', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_frontend', github: 'solidusio/solidus', glob: '**/*.gemspec'
+  gem 'solidus_sample', github: 'solidusio/solidus', glob: '**/*.gemspec'
```
Comment (gsmendoza) [inline Gemfile:10]: The glob format is a workaround for this issue: "Bundler cannot add multiple gems from the same git source" (https://github.com/rubygems/bundler/issues/7346).

**Pull request merged**